robolectric/src/test/java/org/robolectric/shadows/ShadowRenderNodeTest.java

```diff
@@ -0,0 +1,164 @@
+package org.robolectric.shadows;
+
+import android.app.Activity;
+import android.os.Build;
+import android.view.RenderNode;
+import android.view.View;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.robolectric.Robolectric;
+import org.robolectric.TestRunners;
+import org.robolectric.annotation.Config;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+@RunWith(TestRunners.MultiApiWithDefaults.class)
+@Config(sdk = {
+    Build.VERSION_CODES.LOLLIPOP,
+    Build.VERSION_CODES.LOLLIPOP_MR1,
+    Build.VERSION_CODES.M
+})
+public class ShadowRenderNodeTest {
```

Comment:
I decided to add the tests to ShadowViewTest, since this is for a hidden class. When I did this I was able to expose the underlying bug, i.e: that View.setRotation(10f); View.getRotation() != 10f on API 21+ (but it passes on API < 21) and I like this better as i was testing the Android API.

https://github.com/robolectric/robolectric/pull/2309/files